docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
<p>here the bullet points are off, perhaps some spacing or missing enters?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>